tests/test_mixture.py

```diff
@@ -725,7 +727,7 @@ def test_switch_mixture():
     i_vv = I_rv.clone()
     i_vv.name = "i"
 
-    Z1_rv = at.switch(I_rv, X_rv, Y_rv)
+    Z1_rv = op(I_rv, X_rv, Y_rv)
```

Comment:
The added test is identical to the one added in the `Switch` mixture PR. If this test captures what was needed in this PR, would there be a better name than `op` here?

Reply:
`op` works.